pkg/distributor/distributor_test.go

```diff
@@ -1524,7 +1606,7 @@ func TestDistributor_Push_CountDroppedNativeHistograms(t *testing.T) {
 	}
 }
 
-func TestDistributor_SampleDuplicateTimestamp(t *testing.T) {
+func TestDistributor_ValidateSeries(t *testing.T) {
```

Comment:
After fiddling with this test, and realizing I need to make my change in `TestDistributor_Push` instead due to the former only testing `validateSeries`, I think the test's name should reflect what it tests.